lib/signaling/v2/remoteparticipant.js

```diff
@@ -95,7 +96,11 @@ class RemoteParticipantV2 extends RemoteParticipantSignaling {
    */
   addTrack(track) {
     super.addTrack(track);
-    this._getTrackReceiver(track.id).then(track.setTrackTransceiver.bind(track));
+    // NOTE(syerrapragada): DataChannels are rejected in Edge so there
+    // is no trackReceiver for corresponding DataTrackPublication
+    if (guessBrowser() !== 'edge' || track.kind !== 'data') {
```

Comment:
@syerrapragada Can you print a warning to the console once to mention that you won't be able to subscribe to RemoteDataTracks on Edge?

```js
if (guessBrowser() === 'edge' && track.kind === 'data') {
  warnOnce('DataTracks are not supported in Edge. So you won't be able to '
    + 'subscribe to RemoteDataTracks.');
  return;
}
this._getTrackReceiver(track.id).then(track.setTrackTransceiver.bind(track));
```